doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -83,11 +83,11 @@ The vulnerability management process for deployers has at its core the collation
 
 The first of these must be collected by the deployer, while the latter two most often originate from the product Supplier.
 Managing this information is generally called asset management.
-The relationship between SSVC and asset management is discussed further in (Relationship to asset management)[#relationship-to-asset-management].
+The relationship between SSVC and asset management is discussed further in (Relationship to asset management) [#relationship-to-asset-management].
```

Comment:
I think this space breaks the cross reference markdown syntax, doesn't it?

Reply:
I don't know.  Let me take it out.  I don't want to break anything.